packages/salesforcedx-utils-vscode/src/cli/forceOrgList.ts

```diff
@@ -0,0 +1,63 @@
+import { SfdxCommandBuilder } from './commandBuilder';
```

Comment:
I added this command in salesforce-utils-vscode instead of salesforcedx-vscode-core because I am not directly exposing the command to users.